### PR TITLE
bwm-ng: update to 0.6.2.

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

**Comment by Johnnynator** [comment]
Cross seems to be broken.

**Comment by Chocimier** [comment]
Cross never worked and error message says nothing helpful, so I marked as nocross now.

File: srcpkgs/bwm-ng/template [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 pkgname=bwm-ng
-version=0.6.1
-revision=2
+version=0.6.2
+revision=1
 build_style=gnu-configure
+hostmakedepends="automake"
 makedepends="ncurses-devel"
-short_desc="A console-based live network and disk io bandwidth monitor"
+short_desc="Console-basedlive network and disk io bandwidth monitor"
```
Comment (Hoshpak) [inline srcpkgs/bwm-ng/template]: There's a space missing here

**Comment by Chocimier** [comment]
> There's a space missing here
fixed

**Pull request merged**